objects/oBoss1Head.object.gmx

```diff
@@ -142,6 +207,7 @@ alarm[1] = 5;
             <string>if (state != 100) {
     wall1.x = edge_l - 64;
     wall2.x = edge_r + 32;
+    
     with (oCharacter) {
```

Comment:
This is an incredibly minor nitpick that doesn't really matter in this case, but for future reference: the `other` keyword can be used in `with ()` statements to refer to this instance that is executing the `with ()` code. This allows for quicker coding (less problems with renames too!) and safer references to multiple instances in a room (irrelevant here).